config_panel.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "Synapse configuration panel",
+    "version": "0.1",
+    "panel": [{
+        "name": "Synapse config",
+        "id": "synapse_config",
+        "sections": [{
+            "name": "Synapse server configuration",
+            "id": "server_config",
+            "options": [{
+                "name": "Server statistics",
+                "id": "server_statistics",
+                "type": "bool",
+                "help": "True to send anonymous statistics about synapse to improve the performances",
+                "default": false
+            }]
+        }]
+    },{
+        "name": "Package config",
+        "id": "package_config",
+        "sections": [{
+            "name": "Synapse server configuration",
+            "id": "package_config",
+            "options": [{
+                "name": "Backup before upgrade",
+                "id": "backup_before_upgrade",
+                "type": "bool",
+                "help": "True to do a backup before every upgrade",
+                "default": false
```

Comment:
I guess the backup wouldn't be so huge, considering that the default value is false. Can you confirm that ?

Reply:
>I guess the backup wouldn't be so huge

Yes and no, many people said that the database backup is hudge (more than maybe 500Mo).

Can you explain what mean exactly this default value, normally the actuall value is returned by the config script ? And the default value is defined by example in the install script.

Reply:
@Psycojoker could certainly answer this question, I don't really know how this default value will be used.

Reply:
> Can you explain what mean exactly this default value, normally the actuall value is returned by the config script ?

It's the value return if the config script doesn't set any value for it.